tests/resources/transaction/test_mining.py

```diff
@@ -39,7 +39,7 @@ def test_get_block_template_with_address(self):
                 'height': 1,
                 'min_height': 0,
                 'first_block': None,
-                'feature_activation_bit_counts': [0, 0, 0, 0]
+                'feature_activation_bit_counts': None
```

Comment:
Why has is changed to `None`?

Reply:
Because of what was described in https://github.com/HathorNetwork/hathor-core/pull/785#discussion_r1377048608. Since this metadata value is not pre-calculated anymore, but instead lazily calculated (and then stored), its value is now initialized with `None`. This metadata is only internal and is irrelevant for the API being tested there.

Reply:
Got it. Thanks!